react/src/views/BoxEdit/components/BoxEdit.tsx

```diff
@@ -11,11 +11,7 @@ import {
 } from "@chakra-ui/react";
 import { Select } from "chakra-react-select";
 
-import {
-  BoxByLabelIdentifierAndAllProductsQuery,
-  ProductGender,
-  UpdateLocationOfBoxMutation,
```

Comment:
why did you take this one out?

Reply:
@HaGuesto Since `BoxByLabelIdentifierAndAllProductsQuery `contains the types which are needed for the form values, it was removed